tests/Feature/AuthenticationTest.php

```diff
@@ -24,14 +24,14 @@ public function test_users_can_authenticate_using_the_login_screen()
 
         $response = $this->post('/login', [
             'email' => $user->email,
-            'password' => 'password',
+            'password' => '123456',
         ]);
 
         $this->assertAuthenticated();
         $response->assertRedirect(RouteServiceProvider::HOME);
     }
 
-    public function test_users_can_not_authenticate_with_invalid_password()
+    public function test_users_can_not_authenticate_with_invalid_password() // Test fails with current test build
```

Comment:
This seems to be working on my end ;D

Reply:
It was initially throwing a fail when it should have passed, but I discovered it was because the test was looking for the wrong password (initially "password" but it was changed to "123456")